programs/drift_vaults/src/state/vault_depositor.rs

```diff
@@ -236,6 +236,14 @@ impl VaultDepositor {
         vault: &mut Vault,
         now: i64,
     ) -> Result<()> {
+        validate!(
+            vault.max_tokens == 0 || vault.max_tokens < vault_equity.safe_add(amount)?,
```

Comment:
looks like this only works if the accepted vault's deposit token is the same as vault_equity, might be good to just enforce that spot_market_index must be 0?

Reply:
vault_equity is denominated in the spot market index of the vault